projects/go-style-guide/src/app/features/standards/components/colors/colors.component.html

```diff
@@ -88,6 +88,42 @@ <h1 class="go-heading-5">Galaxy Blue</h1>
     </go-card>
   </section>
 
+  <section id="go-color-positive" class="go-column go-column--100">
+    <go-card>
+      <ng-container go-card-header>
+        <h1 class="go-heading-5">Earth Green</h1>
+      </ng-container>
+      <ng-container go-card-content>
+        <section class="go-container">
+          <div class="go-column go-column--50">
+            <p class="go-body-copy">
+              Earth Green should be used to indicate a positive event (ex: form completion), a positive status (ex: indicator),
+               or to show a numerical positive (ex: currency). <br/>
+              <caption>Base SASS variable: $ui-color-positive</caption>
+            </p>
+            <div class="go-color-items">
+              <div class="go-color-items__item">
+                <span class="overline">#1D8915</span>
```

Comment:
Should we be pulling these values from the component.ts file? I also wonder if maybe pulling these values out into a  colors config file might be valuable and then exporting them as a part of goponents to be used in the js files if we ever need them, but that's a conversation for another day.

Reply:
Confirmed that this is not the case actually. We're good as is.